.nvmrc

```diff
@@ -1 +1 @@
-v14
+v18
```

Comment:
I think we should stick to Node 16 until the extension upgrades to 18.

Reply:
Fair enough; will address!